### ISSUE TITLE
Testing GitHub Issues for Blogs

### ISSUE THREAD

**Issue by IlyasOsman** [kuasakenya/kuasa-front#1]
# Steps to Add a Blog Post via GitHub Issues 

### 1. Create a New Issue
- Navigate to the **Issues** tab and click on **New Issue**.

### 2. Title Your Blog Post
- Give your issue a descriptive and engaging title that reflects your blog post's content.

### 3. Write Your Blog Post
- Use **Markdown** to format your blog post in the issue description:
  - Add **headings**, **paragraphs**, **lists**, **images**, **links**, and **code snippets**.
  - You can even embed **audio** or **video** content.

**Example**:
```markdown
# My Blog Subheading/Title
Some introductory paragraph.

## Subheading 1
Here’s a detailed paragraph with useful information.

### Subheading 2
- Bullet point list item
- Another item in the list

[External Resource Link](https://example.com)

```python
# Sample Python code snippet
def hello():
    print("Hello, World!")
```
### 4. Label the Issue as 'Published'
- Once the blog post is ready for release, assign the **'Published'** label.
  - Optionally, apply other relevant labels or tags to categorize your blog (e.g., `Design`, `Propulsion`, `Avionics`, `Structures`).

### 5. Preview Your Blog Post
- Before submitting, use the **Preview** tab to ensure the formatting and content are correct.

### 6. Submit the Issue
- Once you’re satisfied, click **Submit new issue** to publish your blog post.
